documents/src/pages/elements/datetime-picker.md

```diff
@@ -343,6 +344,287 @@ section {
 </ef-datetime-picker>
 ```
 
+## Custom cells
+The datetime-picker allows you to customise any cell on the calendar using slots. 
+Slot names must begin with either `from-` or `to-` as a prefix, indicating which side of the calendar you want to customise. 
+After the prefix, you should specify the date, month, or year as `yyyy-MM-dd`, `yyyy-MM`, or `yyyy` respectively to indicate the specific dates, months, or years that you want to customise. If the prefix is not defined, the slot will automatically cascade to the from calendar.
+
+::
```

Comment:
Calendar overlay covers most of the list of Germany public holidays. Maybe the list should be placed above of datetime-picker instead?
![image](https://github.com/Refinitiv/refinitiv-ui/assets/117723407/8074eeb6-dc29-4800-ad88-85e45fc68ac8)

Reply:
I already swap the list to right side.